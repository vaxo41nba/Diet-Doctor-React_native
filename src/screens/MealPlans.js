import React from 'react';
import {StyleSheet, View} from 'react-native';

export default function MealPlans() {
  return (
    <>
      <View style={{height: 110}} />
      <View style={styles.block} />
    </>
  );
}

const styles = StyleSheet.create({
  block: {
    flex: 1,
    backgroundColor: 'white',
    borderTopRightRadius: 40,
    paddingHorizontal: 10,
    paddingTop: 40,
  },
});
