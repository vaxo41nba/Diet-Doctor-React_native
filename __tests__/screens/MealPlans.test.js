import React from 'react';
import renderer from 'react-test-renderer';
import MealPlans from '../../src/screens/MealPlans';

test('renders correctly', () => {
  const tree = renderer.create(<MealPlans />).toJSON();
  expect(tree).toMatchSnapshot();
});
