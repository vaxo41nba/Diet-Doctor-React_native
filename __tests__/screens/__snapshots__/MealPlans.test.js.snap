// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
[
  <View
    style={
      {
        "height": 110,
      }
    }
  />,
  <View
    style={
      {
        "backgroundColor": "white",
        "borderTopRightRadius": 40,
        "flex": 1,
        "paddingHorizontal": 10,
        "paddingTop": 40,
      }
    }
  />,
]
`;
